Okta/OktaOidcHTTPProtocol.h

```diff
@@ -0,0 +1,16 @@
+//
+//  OktaOidcHTTPProtocol.h
+//  okta-oidc
+//
+//  Created by Lihao Li on 8/27/20.
+//  Copyright © 2020 Okta. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol OktaOidcHTTPProtocol <NSObject>
```

Comment:
If it's a delegate, I would prefer OktaOidcHttpDelegate. Alternatively, you could use the approach in OktaDeviceSDK where an HTTPClient is passed in and then used. In that case protocol is fine 👍

Reply:
Probably `OktaOidcHTTPNotifyDelegate`